securedrop_client/storage.py

```diff
@@ -276,24 +276,30 @@ def update_sources(
     for source in remote_sources:
         if source.uuid in local_sources_by_uuid:
 
-            # If source UUID has been flagged as having been locally updated, wait
-            # til next sync to update it. The list of skip_uuids is managed by the
-            # parent method and purged after each sync.
+            # Update an existing record.
```

Comment:
This is a change I made in #1432 that I want to revert/rework. Basically, the plan here (and what I should have done all along) is to update the source record every sync, no matter what, with all attributes except the `document_count`, which is updated conditionally.

Reply:
@rocodes To which extent would it be accurate to broadly describe the idea as: "record locally the state of the remote, then figure out the implications for the local state"? (Checking if I'm understanding our approach.)